Mesh_3/include/CGAL/Mesh_3/parameters_defaults.h

```diff
@@ -23,14 +23,16 @@
 
 #include <CGAL/Mesh_3/sliver_criteria.h>
 
+// see also default_values_for_mesh_3 namespace
+// in CGAL/STL_Extension/internal/mesh_option_classes.h
+
 namespace CGAL {
 namespace parameters { namespace default_values_for_mesh_3 {
 
 // exude_mesh_3
 const double exude_sliver_bound = 0.;
 
 // perturb_mesh_3
-const double perturb_sliver_bound = 0.;
```

Comment:
I did the minimum so that it can compile solving dependencies without making STL_Extension depends on Mesh_3. We can move it there but where do we stop?

Reply:
I'd say the default parameters should all be at the same place, otherwise it's really tedious to navigate code.